src/Entity/Comment.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace App\Entity;
+
+use Cycle\Annotated\Annotation\Column;
+use Cycle\Annotated\Annotation\Entity;
+use Cycle\Annotated\Annotation\Relation\BelongsTo;
+use Cycle\Annotated\Annotation\Table;
+use Cycle\Annotated\Annotation\Table\Index;
+use DateTimeImmutable;
+
+/**
+ * @Entity(
+ *     mapper="App\Mapper\CommentMapper",
+ *     constrain="App\Constrain\CommentPublic"
+ * )
+ * @Table(
+ *     indexes={
+ *         @Index(columns={"public","publishedAt"})
+ *     }
+ * )
+ */
+class Comment
+{
+    /**
+     * @Column(type="primary")
+     * @var int
+     */
+    private $id;
```

Comment:
How about types for entities?

Reply:
Not all types can be affixed. But here its possible, because Cycle will convert a string value to an integer